lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -544,7 +540,7 @@ def _get_usage_from_flow_stat(self, flow_stats):
         Update the rule record map with the flow stat and return the
         updated map.
         """
-        current_usage = defaultdict(RuleRecord)
+        current_usage = defaultdict(lambda: defaultdict(RuleRecord))
```

Comment:
what does this do?

Reply:
Defined list of defaultdict for index 0 belongs to 5g flows and for index 1 belongs to 4g flows